### DISCUSSION TITLE
Rediriger vers un autre synapps

### DISCUSSION THREAD

**Discussion by SeddikRebbaj** [witsa/synapps#691] [Q&A]
Bonjour, 

Je vous contacte pour savoir comment on fait pour faire un bouton qui permet lors d'un clique de redirigé vers un autre SynApps et pas une scène je souhaite basculer du synapps bois au synnaps vug à l'aide du bouton retour .
![tous les synapps](https://github.com/witsa/synapps/assets/151936804/c3ca46a0-3c7d-4905-9458-d287ae222ed3)
![bouton retour](https://github.com/witsa/synapps/assets/151936804/b24db738-ffb3-4bfb-a9a6-43fdf2489e58)

Cordialement

**Comment by ArmandGC** [comment]
Salut,
Tu peux effectivement le faire (en tout cas ça marche pour moi en local donc a tester sur un WIT)
![synapps-studio_zvcwCsw81b](https://github.com/witsa/synapps/assets/134411061/05b7bd1c-cd0b-4031-b66e-7928aba1442a)
sur le bouton tu as l'additionnel "lien" qui est liée au SID pour récupérer L'identité de la session ouverte et il n'y a plus qu'a modifier dans le OnReadTransform
pour le vug : return "http://192.168.145.135/WSID"+context.value+"/easy/synapps/bois";
pour le bois : return "http://192.168.145.135/WSID"+context.value+"/easy/synapps/vug";

`SYNAPPS-STUDIO-ACTOR|{"type":"input/button","key":"LienVersVug","properties":{"content":"<button style=\"display: contents; color: {{Color}};\" onclick=\"window.location.href = '{{Lien}}';\"><i class=\"icon-{{icon}}\"></i> {{text}}</button>\n"},"additionalDefs":{"icon":{"type":"icon","label":"Icône","description":"Web Hosting Hub Glyphs"},"text":{"type":"text","label":"Texte"},"Lien":{"type":"text"},"Color":{"type":"color"}},"additionals":{"icon":"fire","text":"Vug","Color":"rgba(48, 65, 193, 1)"},"bindings":{"additionals.Color":{"sourceType":"color","path":"themeColor2"},"additionals.Lien":{"sourceType":"session","path":"sid"}},"events":{"additionals/Lien/binding/onReadTransform":["return \"http://192.168.145.135/WSID\"+context.value+\"/easy/synapps/vug\";"]}}`

Je l'ai tester sur Redy PC avec deux de mes projets et ça semble bien fonctionner.
